TestingWebAPI/Controllers/NationalizeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Core.Interfaces.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace TestingWebAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class NationalizeController : ControllerBase
+    {
+        private readonly IEmployeeService _employeeService;
+        private readonly INationalizeService _nationalizeService;
+
+        public NationalizeController(IEmployeeService employeeService, INationalizeService nationalizeService)
+        {
+            _employeeService = employeeService;
+            _nationalizeService = nationalizeService;
+        }
+
+        [HttpGet("{id}/nationalities")]
+        public async Task<ActionResult> GetEmployeeNationalities(int id)
```

Comment:
we agreed that type needs to be specified here, so that swagger can document the sample response